src/app/my-profile/components/my-profile-edit/my-profile-edit.component.html

```diff
@@ -0,0 +1,98 @@
+<div
+  *erzLet="{ formGroup: formGroup$ | async, student: student$ | async } as data"
+  class="erz-container erz-container-padding-y"
+>
+  <form
+    *ngIf="data.formGroup"
+    [formGroup]="data.formGroup"
+    (ngSubmit)="onSubmit()"
+  >
+    <p>{{ 'my-profile.edit.address-changes-remark' | translate }}</p>
+    <address class="text-muted">
```

Comment:
Ein Detail: Wir verwenden überall (also in der gesamten Applikation) ein bisschen andere Grautöne, wenn ich das richtig gesehen habe. Vielleicht könnten wir das noch vereinheitlichen.